### ISSUE TITLE
New menu 

### ISSUE THREAD

**Issue by marinofaggiana** [nextcloud/ios#1106]
@immortal79 Ipad pro 11"

<img width="830" alt="Schermata 2020-01-25 alle 11 45 53" src="https://user-images.githubusercontent.com/8616947/73119921-86446780-3f68-11ea-85ec-2820f07846ff.png">

**Comment by marinofaggiana** [comment]
And strange effect with iPhone 11 landscape <> portrait and I think that is better hide the scroolbar in the tableview menu...

**Comment by marinofaggiana** [comment]
No good with small iPhone 6s iOS 10 (landscape <> portrait), and issue after delete a record with swipe

**Comment by marinofaggiana** [comment]
![Simulator Screen Shot - iPhone 11 - 2020-01-26 at 10 20 47](https://user-images.githubusercontent.com/8616947/73133215-d6353400-4025-11ea-8a9d-60d80d1ead7a.png)

**Comment by immortal79** [comment]
Will fix ASAP

**Comment by marinofaggiana** [comment]
@immortal79 look:

![20-01-28 09-33-45 8088](https://user-images.githubusercontent.com/8616947/73247604-8c795480-41b1-11ea-8dc3-3d706fa23163.jpg)

**Comment by immortal79** [comment]
Is this what you want or what it looks like in your simulator ?
If this is what you want the view to look like, it's actually a very simple solution to implement and I can do it right now.

**Comment by marinofaggiana** [comment]
Yes, for me this is better (simply) and will adapt to the next bar for iPad with the [+] on center of TabBar

**Comment by marinofaggiana** [comment]
> Is this what you want or what it looks like in your simulator ?
> If this is what you want the view to look like, it's actually a very simple solution to implement and I can do it right now.
> 
> Edit: didn't see your edit, I can do this 👍

yes, look dropbox

![20-01-28 09-41-35 8089](https://user-images.githubusercontent.com/8616947/73248068-8c2d8900-41b2-11ea-926b-1b8a27807c26.png)

**Comment by marinofaggiana** [comment]
Whats do you think?

**Comment by immortal79** [comment]
Ok for me, I start working on it :)

**Comment by marinofaggiana** [comment]
After https://github.com/nextcloud/ios/pull/1116: 

![Simulator Screen Shot - iPhone 11 - 2020-01-28 at 10 26 49](https://user-images.githubusercontent.com/8616947/73251178-ba15cc00-41b8-11ea-99e0-bcadab84357c.png)

bug : from portrait to landscape with several item is not possible move to te first item ... test it
design : I don't like the margin on safe area ... left right

**Comment by marinofaggiana** [comment]
@immortal79  it would be nice to add : 

![Schermata 2020-01-28 alle 10 31 52](https://user-images.githubusercontent.com/8616947/73251688-b3d41f80-41b9-11ea-959e-3037dd187726.png)

**Comment by marinofaggiana** [comment]
Whats do you think? ( do we do a test? )

**Comment by immortal79** [comment]
> After #1116:
> 
> ![Simulator Screen Shot - iPhone 11 - 2020-01-28 at 10 26 49](https://user-images.githubusercontent.com/8616947/73251178-ba15cc00-41b8-11ea-99e0-bcadab84357c.png)
> 
> bug : from portrait to landscape with several item is not possible move to te first item ... test it
> design : I don't like the margin on safe area ... left right

I can add the top grabber.
But I don't know about the margin I do a test + screenshot and come back to you.

**Comment by marinofaggiana** [comment]
https://github.com/nextcloud/ios/pull/1118

Great ! What's do you think?

@immortal79  Bug: iPhone open menu in Portrait, move in landscape, with the below gesture for close it, it does not close

**Comment by immortal79** [comment]
#1119 should fix it

**Comment by marinofaggiana** [comment]
Bug with iOS 10.3.1 the gesture do not works

**Comment by marinofaggiana** [comment]
@immortal79 I think that you can replace the other menu sheeeeeet

**Comment by marinofaggiana** [comment]
BUG: open menu, move with finger a little down and then move up, compare a line white below.
BUG: order for size do not works

**Comment by immortal79** [comment]
Will look into the first bug.
Didn't understand the second one ?

**Comment by marinofaggiana** [comment]
The menu "order A/Z, Date, size": Order for size do not works :-)

**Comment by immortal79** [comment]
Oh ok now I get you, will fix that too.

**Comment by marinofaggiana** [comment]
@immortal79 Bug: Main menu: Open in... crashed

**Comment by marinofaggiana** [comment]
@immortal79 in AppDelegate.m/ - (void)handleTouchTabbarCenter:(id)sender

UIViewController *vc = _activeMain.splitViewController.viewControllers[0];

now this vc is a NavigationController ... it can be a problem ?
